sql/courses.sql

```diff
@@ -80,3 +80,13 @@ CREATE TABLE scheduler_sections (
         REFERENCES sections(course_id, section_number)
         ON DELETE CASCADE
 );
+
+CREATE TABLE scheduler_notes (
+    user_id         INT      NOT NULL,
+    course_id       INT      NOT NULL,
+    notes	    TEXT     DEFAULT NULL,
+    PRIMARY KEY (user_id, course_id),
+    FOREIGN KEY (user_id, course_id) 
+	REFERENCES scheduler_sections(user_id, course_id)
```

Comment:
`(user_id, course_id)` is not a unique key for `scheduler_sections` since a user can add multiple sections of the same course